integrationTesting/tests/organize/journeys/instance-detail/display.spec.ts

```diff
@@ -27,24 +27,23 @@ test.describe('Journey instance detail page', () => {
       ClarasOnboarding
     );
 
-    await page.goto(appUri + '/organize/1/journeys/1/1');
-
-    // Check that the title is visible in the right place
-    expect(
-      await page
-        .locator(
-          `[data-testid=page-title]:has-text("${ClarasOnboarding.title}")`
-        )
-        .count()
-    ).toEqual(1);
-
-    // Check that the title is also in the breadcrumbs
-    expect(
-      await page
-        .locator(
-          `[aria-label=breadcrumb]:has-text("${ClarasOnboarding.title}")`
-        )
-        .count()
-    ).toEqual(1);
+    const title = page.locator(
+      `[data-testid=page-title]:has-text("${ClarasOnboarding.title}")`
+    );
+    const breadcrumbTitle = page.locator(
+      `[aria-label=breadcrumb]:has-text("${ClarasOnboarding.title}")`
+    );
+
+    await Promise.all([
+      page.goto(appUri + '/organize/1/journeys/1/1'),
+      title.waitFor({ state: 'visible' }),
+      breadcrumbTitle.waitFor({ state: 'visible' }),
+    ]);
+
+    const numTitles = await title.count();
+    const numBreadcrumbTitles = await breadcrumbTitle.count();
+
+    expect(numTitles).toEqual(1);
+    expect(numBreadcrumbTitles).toEqual(1);
```

Comment:
As an example, I think the important part of the fix here is the added uses of `waitFor()`, not the `Promise.all()`. I'm pretty sure you could do this instead?

```tsx
await page.goto(appUri + '/organize/1/journeys/1/1');
await title.waitFor({ state: 'visible' });
await breadcrumbTitle.waitFor({ state: 'visible' });

const numTitles = await title.count();
const numBreadcrumbTitles = await breadcrumbTitle.count();

expect(numTitles).toEqual(1);
expect(numBreadcrumbTitles).toEqual(1);
```

Do you agree that this pattern would make the code more readable?

Reply:
You're 100% correct here I think. Reckon I've accidentally picked this up from those other ones you mentioned and not really thought much about it. Crossed my mind briefly at one point I think, but guess I got a little lost in the fun here. Will make this change real quick, thanks for raising this!